src/Steps/index.tsx

```diff
@@ -23,6 +29,9 @@ export default function Steps(props: StepsProps) {
       dotSet.add(mark.value);
     });
 
+    //Fill marksValue
+    marksValueRef.current = Array.from(dotSet);
```

Comment:
useEffect for `stepDots` change, or:

 const { stepDots,  marksValueRe }= React.useMemo(...)

Reply:
ok thanks！